tests/template_tests/test_custom.py

```diff
@@ -96,130 +100,115 @@ def test_simple_tag_registration(self):
 
     def test_simple_tag_missing_context(self):
         # The 'context' parameter must be present when takes_context is True
-        six.assertRaisesRegex(self, template.TemplateSyntaxError,
-            "'simple_tag_without_context_parameter' is decorated with takes_context=True so it must have a first argument of 'context'",
-            template.Template, '{% load custom %}{% simple_tag_without_context_parameter 123 %}')
-
-    def test_inclusion_tags(self):
-        c = template.Context({'value': 42})
-
-        t = template.Template('{% load inclusion %}{% inclusion_no_params %}')
-        self.assertEqual(t.render(c), 'inclusion_no_params - Expected result\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_one_param 37 %}')
-        self.assertEqual(t.render(c), 'inclusion_one_param - Expected result: 37\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_explicit_no_context 37 %}')
-        self.assertEqual(t.render(c), 'inclusion_explicit_no_context - Expected result: 37\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_no_params_with_context %}')
-        self.assertEqual(t.render(c), 'inclusion_no_params_with_context - Expected result (context value: 42)\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_params_and_context 37 %}')
-        self.assertEqual(t.render(c), 'inclusion_params_and_context - Expected result (context value: 42): 37\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_two_params 37 42 %}')
-        self.assertEqual(t.render(c), 'inclusion_two_params - Expected result: 37, 42\n')
+        msg = ("'simple_tag_without_context_parameter' is decorated with "
+              "takes_context=True so it must have a first argument of 'context'")
+        contents = '{% load custom %}{% simple_tag_without_context_parameter 123 %}'
+        six.assertRaisesRegex(self, TemplateSyntaxError, msg, Template, contents)
```

Comment:
I think you can use `with self.assertRaisesMessage` equivalently here (context manager form is much easier to read IMO)